benchmark-service/benchmark/utils.py

```diff
@@ -69,12 +74,11 @@ def get_data_as_string(self, file_path):
         bucket = self.storage_client.get_bucket(file_info['bucket'])
         path = file_info['path']
         try:
-            # get bucket data as blob
             blob = bucket.blob(path)
-            # convert to string
             data = blob.download_as_string()
-        except Exception as e:
-            raise NameError() from e
+        except google.api_core.exceptions.NotFound as e:
+            log.exception(f'error while reading file {file_path}: {e}')
+            data = ""
```

Comment:
It would probably be better to return None here. I'm not sure where it's used though to know for sure what is the right interface.

Reply:
I checked where it's used and I agree that None is better

Reply:
also about the question above yes lookup.html is a copy of index.html from before